dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
       },
       {
         "value": "650QB",
-        "label": "EAGLE STREET VA CLINIC"
+        "label": "Eagle Street VA Clinic"
       },
       {
         "value": "650",
-        "label": "PROVIDENCE VAMC"
+        "label": "Providence VA Medical Center"
       }
     ],
     "DC": [
       {
         "value": "688GB",
-        "label": "SOUTHEAST WASHINGTON VA CLINIC"
+        "label": "Southeast Washington VA Clinic"
       },
       {
         "value": "688QA",
-        "label": "FRANKLIN STREET VA CLINIC"
+        "label": "Franklin Street VA Clinic"
       },
       {
         "value": "688",
-        "label": "WASHINGTON VA MEDICAL CENTER"
+        "label": "Washington VA Medical Center"
       }
     ],
     "CT": [
       {
         "value": "689A4",
-        "label": "NEWINGTON"
+        "label": "Newington VA Clinic"
       },
       {
         "value": "689GA",
-        "label": "WATERBURY"
+        "label": "Waterbury VA Clinic"
       },
       {
         "value": "689GB",
-        "label": "STAMFORD VA PRIMARY CARE CTR"
+        "label": "Stamford VA Clinic"
       },
       {
         "value": "689GC",
-        "label": "WINDHAM CBOC"
+        "label": "Willimantic VA Clinic"
       },
       {
         "value": "689GD",
-        "label": "WINSTED"
+        "label": "Winsted VA Clinic"
       },
       {
         "value": "689GE",
-        "label": "DANBURY"
+        "label": "Danbury VA Clinic"
       },
       {
         "value": "689HC",
-        "label": "NEW LONDON"
+        "label": "John J. McGuirk Department of Veterans Affairs Outpatient Clinic"
       },
       {
         "value": "689HK",
         "label": "WEST HAVEN MORC"
       },
       {
         "value": "689",
-        "label": "VA CONNECTICUT HEALTHCARE SYSTEM - WEST HAVEN DIVISION"
+        "label": "West Haven VA Medical Center"
       }
     ],
     "UT": [
       {
         "value": "660GJ",
-        "label": "WESTERN SALT LAKE VA CLINIC"
+        "label": "Western Salt Lake VA Clinic"
       },
       {
         "value": "660GB",
-        "label": "OGDEN VA CLINIC"
+        "label": "Ogden VA Clinic"
       },
       {
         "value": "660GD",
-        "label": "ROOSEVELT VA CLINIC"
+        "label": "Roosevelt VA Clinic"
       },
       {
         "value": "660GE",
-        "label": "OREM VA CLINIC"
+        "label": "Orem VA Clinic"
       },
       {
         "value": "660GG",
-        "label": "ST. GEORGE VA CLINIC"
+        "label": "St. George VA Clinic"
       },
       {
         "value": "660QC",
-        "label": "WEBER COUNTY VA CLINIC"
+        "label": "Weber County VA Clinic"
       },
       {
         "value": "660",
-        "label": "GEORGE E. WAHLEN VAMC"
+        "label": "George E. Wahlen Department of Veterans Affairs Medical Center"
       }
     ],
     "PR": [
       {
         "value": "672B0",
-        "label": "PONCE OPC"
+        "label": "Eurípides Rubio Department of Veterans Affairs Outpatient Clinic"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.

Reply:
Confirmed this is the same facility, since this is in Ponce, PR (https://www.va.gov/find-locations/facility/vha_672B0)